staging/src/k8s.io/apiserver/pkg/quota/v1/resources.go

```diff
@@ -215,15 +215,15 @@ func Difference(a []corev1.ResourceName, b []corev1.ResourceName) []corev1.Resou
 	return result
 }
 
-// IsZero returns true if each key maps to the quantity value 0
-func IsZero(a corev1.ResourceList) bool {
```

Comment:
add the new RemoveZeros method, but leave this method alone (it could be used by other callers)